### PR TITLE
Adds savage-banana-soup

### PR DESCRIPTION
**Pull request by PatrickLohan**
I hope all is good. Love VueJS, keep it rocking!

### CODE REVIEW

File: src/recipes/savage-banana-soup.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "directions": [
+        "Put spinach/kale in bottom of smoothie maker.",
+        "Next is the ginger, peanut butter and flax/linseed.",
+        "Bananas and ice go on top, followed by the milk.",
+        "Blend for enough time so the ice is crushed up.",
+        "Enjoy! Go get 'em, tiger!'"
+    ],
+    "image": "savage-banana-soup.jpg",
+    "source": "<url if available>"
```
Comment (alfg) [inline src/recipes/savage-banana-soup.json]: This line can be removed if a url is not available. Thanks!

**Comment by PatrickLohan** [comment]
Whoops, knew I'd made a mistake. Forgot the blueberries!

File: src/recipes/savage-banana-soup.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "description": "A smoothie that bites back!",
+    "github": "PatrickLohan",
+    "ingredients": [
+        "50g Spinach or Kale",
+        "10g Ground Ginger",
+        "30g Peanut Butter",
+        "10g Ground Flax/Linseed",
+        "200g Banana, ripe",
+        "100g Blueberries",
+        "300ml Almond/Coconut Milk",
```
Comment (alfg) [inline src/recipes/savage-banana-soup.json]: Extra comma breaking json. Almost there!
```suggestion
        "300ml Almond/Coconut Milk"
```

Comment (PatrickLohan) [reply src/recipes/savage-banana-soup.json]: Thanks Alfg, always a stray comma/semi-colon ;)

Comment (PatrickLohan) [reply src/recipes/savage-banana-soup.json]: I think I don't need to do anything but click resolve, as you're suggesting changes as maintainer? Pretty new to github, sorry.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**